config/regex.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
     TEMPLATE_NAME: /^[\w/-]+$/,
     // Version can only have up to 2 decimals, like 1.2.3
     VERSION: /^(\d+\.)?(\d+\.)?(\d+)$/,
```

Comment:
I added `VERSION` for template validation, so it should be okay
https://github.com/screwdriver-cd/data-schema/pull/109/files